lib/aws-record/record/item_operations.rb

```diff
@@ -581,19 +576,21 @@ def find_all(keys)
         # Aws::DynamoDB::Client#update_item} call immediately on the table,
         # using the attribute key/value pairs provided.
         #
-        # @param [Hash] opts attribute-value pairs for the update operation you
-        #  wish to perform. You must include all key attributes for a valid
+        # @param [Hash] new_params attribute-value pairs for the update operation
```

Comment:
naming nit: maybe instead of new_params we could just use `attributes`?

Reply:
Unfortunately `attributes` would shadow the [attributes](https://github.com/aws/aws-sdk-ruby-record/blob/92e0070eea16698e23be8e31d3a36e681dc079e9/lib/aws-record/record/attributes.rb#L514C1-L517) method already used below:
https://github.com/aws/aws-sdk-ruby-record/blob/92e0070eea16698e23be8e31d3a36e681dc079e9/lib/aws-record/record/item_operations.rb#L599-L600

I used `new_params` because it was consistent with the `ItemOperations#update`/`ItemOperations#update!` signatures also in this file. But maybe that's confusing since in this particular case it also must include the keys?

Reply:
Nvm - I think `new_params` makes sense for consistency then.